### ISSUE TITLE
Incompatible types in assignment with User subclass

### ISSUE THREAD

**Issue by zacchiro** [typeddjango/django-stubs#174]
Heya, I've this class deriving from the (concrete) User class:
```python
from django.contrib.auth.models import User, UserManager

class DepositClient(User):
    objects = UserManager()
```
mypy with django-stubs 1.1.0 is complaining on the `objects` line that:
```
error: Incompatible types in assignment
(expression has type "UserManager[DepositClient]",
base class "User" defined the type as "UserManager[User]")
```
even if `DepositClient` is a subclass of `User` and the type variable in `Manager` is declared as covariant by django-stubs.

This should work out-of-the-box with django-stubs, right?
Or am I missing something here?

**Comment by mkurnikov** [comment]
This is a case specified in the https://github.com/typeddjango/django-stubs/blob/master/test-data/typecheck/managers/test_managers.yml#L281
It's a mypy limitation, smth around Liskov incompatibility.

**Comment by zacchiro** [comment]
On Mon, Sep 23, 2019 at 09:37:11AM -0700, Maxim Kurnikov wrote:
> This is a case specified in the
> https://github.com/typeddjango/django-stubs/blob/master/test-data/typecheck/managers/test_managers.yml#L281
> It's a mypy limitation, smth around Liskov incompatibility.

So what is the recommended way to go about these?  # type: ignore ?

**Comment by mkurnikov** [comment]
Yes, `# type: ignore` + manual annotation I believe would work (can't test right now).
Until https://github.com/python/mypy/issues/7468 is fixed.
